WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1086,6 +1091,30 @@ import WordPressComAnalytics
         }
     }
 
+    func backgroundFetch(_ completionHandler: ((UIBackgroundFetchResult) -> Void)?) {
+        print("=========>>>>>>>>>>> backgroundFetch")
+        print("=========>>>>>>>>>>> \(String(describing: readerTopic?.title))")
+
+        let lastSeenPostID = (tableViewHandler.resultsController.fetchedObjects?.first as? ReaderPost)?.postID
+
+        syncHelper.backgroundSync(success: { [weak self, weak lastSeenPostID] in
+            let newestFetchedPostID = (self?.tableViewHandler.resultsController.fetchedObjects?.first as? ReaderPost)?.postID
```

Comment:
This is exactly why I don't really like optionals. Do you think it is better if we give it a default value? Something like: 


```        
let lastSeenPostID = (tableViewHandler.resultsController.fetchedObjects?.first as? ReaderPost)?.postID ?? -1
syncHelper.backgroundSync(success: { [weak self, weak lastSeenPostID] in
    let newestFetchedPostID = (self?.tableViewHandler.resultsController.fetchedObjects?.first as? ReaderPost)?.postID ?? -1
    self?.tableView.scrollToRow(at: IndexPath(row: 0, section: 0), at: .top, animated: false)
    if lastSeenPostID == newestFetchedPostID {
```